views/components/footer/_footer.scss

```diff
@@ -0,0 +1,10 @@
+footer {
+	font-size: 0.875rem;
+	display: flex;
+	justify-content: flex-end;
+
+	a {
+		 margin-left: 1rem;
+		 color: hsl(205.7, 6.3%, 56.1%);
+	}
+}
```

Comment:
* color value copied from `lib/styles/index.scss` 😬 
* magic numers copied from elsewhere 🤷

Reply:
I think the magic numbers are ok for now...

Well... they aren't really ok. We need a decent spacing scale. But that's not in the scope of this change, so for this PR they are ok.